transactions/2.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Transactions 2.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Transactions 2.0"
+date: 2020-10-08
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Transactions - Need scope statement
 
 * [Jakarta Transactions 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jta/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Transactions 2.0 Specification Document]() (PDF)
-* [Jakarta Transactions 2.0 Specification Document]() (HTML)
+* [Jakarta Transactions 2.0 Specification Document](./jakarta-transactions-spec-2.0.pdf) (PDF)
+* [Jakarta Transactions 2.0 Specification Document](./jakarta-transactions-spec-2.0.html) (HTML)
 * [Jakarta Transactions 2.0 Javadoc](./apidocs)
-* [Jakarta Transactions 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Transactions 2.0 TCK](https://download.eclipse.org/jakartaee/transactions/2.0/jakarta-transactions-tck-2.0.0.zip)([sig](https://download.eclipse.org/jakartaee/transactions/2.0/jakarta-transactions-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/transactions/2.0/jakarta-transactions-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.jta:jakarta.jta-api:jar:2.0.0](https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/2.0.0-RC2/jakarta.transaction-api-2.0.0-RC2.jar)
+  * [jakarta.jta:jakarta.jta-api:jar:2.0.0](https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/2.0.0/jakarta.transaction-api-2.0.0.jar)
```

Comment:
The maven coordinates should be jakarta.transaction:jakarta.transaction-api:jar:2.0.0
```suggestion
  * [jakarta.transaction:jakarta.transaction-api:jar:2.0.0](https://repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/2.0.0/jakarta.transaction-api-2.0.0.jar)
```

Reply:
I agree, this was an oversight and is wrong as it stands because TCK was with the staged final jar.

@dblevins am I OK to commit the suggestion while the ballot is active?

Reply:
@tomjenkinson As we discussed on the Spec Committee call today, it is okay to make these minor updates to the _index.md file.  Thanks!

Reply:
Thanks, I will commit the suggestion - thanks @hussainnm !